src/Microsoft.AspNetCore.Razor.Language/Resources.resx

```diff
@@ -264,4 +264,34 @@
   <data name="InvalidRazorLanguageVersion" xml:space="preserve">
     <value>Provided value for razor language version is unsupported or invalid: '{0}'.</value>
   </data>
+  <data name="AddTagHelperDirective_StringToken_Description" xml:space="preserve">
+    <value>Represents the tag helper type name and the assembly name to add the tag helpers from. Specify '*' for type name to include all tag helper types from the given assembly.</value>
```

Comment:
`Add tag helpers from the specified type name and assembly name. Specify '*' for the type name to include all tag helper types from the specified assembly.`